scripts/qa_crowbarsetup.sh

```diff
@@ -4208,6 +4208,59 @@ EOF
     exit $ret
 }
 
+function ping_fips
+{
+    local fips=$(openstack ip floating list -f value -c IP)
+    for fip in $fips; do
+        ping -c 1 -w 60 $fip || complain 120 "cannot reach test VM at $fip."
```

Comment:
this is inherited from the tests I did for connectivity loss during the upgrade and it's not really necessary here, although works properly. The command reads "keep sending pings for up to 60 seconds until the IP replies". In practice that means only one ping will be sent should the VM be alive. It could be reduced but don't know what the proper value should be.